cocotb/share/lib/vhpi/VhpiImpl.h

```diff
@@ -224,6 +224,7 @@ class VhpiLogicSignalObjHdl : public VhpiSignalObjHdl {
 
     virtual ~VhpiLogicSignalObjHdl() { }
 
+    using GpiSignalObjHdl::set_signal_value;
```

Comment:
This will cause `set_signal_value(double)` to segfault. Previously, this caused an implicit conversion of a `double` argument` to a `long` and called that overload. I have a Issue/PR coming that will explain and fix the issues with the VHPI and FLI related to overloaded virtual functions.

Reply:
Can you elaborate on why this causes a segfault? Is the issue that `GpiSignalObjHdl::set_signal_value` is pure-virtual? If so, isn't this a compiler error?

> Previously, this caused an implicit conversion of a double argument to a long and called that overload.

I don't agree with that - from what I can tell, overloads are not used to populate the v-table, so this doesn't actually matter for the way we use these objects.

Reply:
You are right, it's only an issue if we call it as a derived object.